frontend/src/components/utilities/cryptography/crypto.ts

```diff
@@ -210,7 +210,14 @@ const decryptSymmetric = ({ ciphertext, iv, tag, key }: DecryptSymmetricProps):
   try {
     plaintext = aes.decrypt({ ciphertext, iv, tag, secret: key });
   } catch (err) {
-    console.log("Failed to perform decryption");
+    console.log("Failed to decrypt with the following parameters", {
```

Comment:
can you please use the logger and not console.log?

Reply:
We don't have a logger on the frontend unfortunately